src/components/AlphanumericInput/AlphanumericInput.svelte

```diff
@@ -2,7 +2,7 @@
   import { _ } from 'svelte-i18n'
   import type { Event } from './utils'
 
-  import { screen } from '../../stores'
+  import { screen } from '../../store'
```

Comment:
Please use a folder called `stores`:

```suggestion
  import { screen } from '../../stores'
```

Reply:
As there is many stores in it, yes :+1:

Reply:
✅